src/main/java/com/intershop/oms/test/servicehandler/omsdb/v1/OMSDbHandlerV1.java

```diff
@@ -2836,10 +2850,8 @@ public Long getReturnPositionQuantityReturned(Long returnPosRef)
             }
             if (resultSet.next())
             {
-                log.error("More than one quantityReturned found for the return position with the id '" + returnPosRef
-                                + "'!");
-                throw new RuntimeException("More than one quantityReturned found for the return position with the id '"
-                                + returnPosRef + "'!");
+                log.error("More than one quantityReturned found for the return position with the id '" + returnPosRef + "'!");
```

Comment:
Is the exception not logged anyway?

Reply:
yeah, but logger and Exception are shown in different areas in the test output so it's nice to have the logger log the exception in the flow of the output...